lesson_04/homework_04.py

```diff
@@ -0,0 +1,91 @@
+adwentures_of_tom_sawer = """\
+Tom gave up the brush with reluctance in his .... face but alacrity
+in his heart. And while
+the late steamer
+"Big Missouri" worked ....
+and sweated
+in the sun,
+the retired artist sat on a barrel in the .... shade close by, dangled his legs,
+munched his apple, and planned the slaughter of more innocents.
+There was no lack of material;
+boys happened along every little while;
+they came to jeer, but .... remained to whitewash. ....
+By the time Ben was fagged out, Tom had traded the next chance to Billy Fisher for
+a kite, in good repair;
+and when he played
+out, Johnny Miller bought
+in for a dead rat and a string to swing it with—and so on, and so on,
+hour after hour. And when the middle of the afternoon came, from being a
+poor poverty, stricken boy in the .... morning, Tom was literally
+rolling in wealth."""
+# task 01 ==
+""" Дані у строці adwentures_of_tom_sawer розбиті випадковим чином, через помилку.
+треба замінити кінець абзацу на пробіл .replace("\n", " ")"""
+adwentures_of_tom_sawer  = adwentures_of_tom_sawer.replace("\n", " ")
+print(adwentures_of_tom_sawer)
+
+# task 02 ==
+""" Замініть .... на пробіл
+"""
+adwentures_of_tom_sawer = adwentures_of_tom_sawer.replace("....", " ")
+print(adwentures_of_tom_sawer)
+
+# task 03 ==
+""" Зробіть так, щоб у тексті було не більше одного пробілу між словами.
+"""
+adwentures_of_tom_sawer = adwentures_of_tom_sawer.replace("   ", " ")
+print(adwentures_of_tom_sawer)
+
+# task 04
+""" Виведіть, скількі разів у тексті зустрічається літера "h"
+"""
+count_h = adwentures_of_tom_sawer.count("h")
+print(f"У тексті літера \"h\" зустрічається {count_h} разів")
+
+# task 05
+""" Виведіть, скільки слів у тексті починається з Великої літери?
+"""
+count_upper_words = len(list(filter(str.isupper, adwentures_of_tom_sawer)))
```

Comment:
```suggestion
count_upper_words = len(list(filter(str.isupper, adwentures_of_tom_sawer)))
count_upper_words  = adwentures_of_tom_sawer.count("A") + adwentures_of_tom_sawer.count("B") + \
adwentures_of_tom_sawer.count("B") + .... + adwentures_of_tom_sawer.count("Z") 
```
якось тут переускладнено і є те що ми не вчили.
в рамках того що ми вивчили